src/Dock.Avalonia.Themes.Default/ToolControl.xaml

```diff
@@ -7,76 +7,73 @@
-                                <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}" />
-                                <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderLowBrush}" />
-                            </Style>
-                        </TabStrip.Styles>
-                        <TabStrip.DataTemplates>
-                            <DataTemplate DataType="Dock:IDockable">
-                                <StackPanel Name="DragTool" id:DockProperties.IsDragArea="True" id:DockProperties.IsDropArea="True" Background="Transparent" Orientation="Horizontal" Spacing="2">
-                                    <TextBlock Text="{Binding Title}" Classes="drag" Margin="2" />
-                                </StackPanel>
-                            </DataTemplate>
-                        </TabStrip.DataTemplates>
-                    </TabStrip>
-                    <Border Classes="panel" ZIndex="-1" Background="{DynamicResource ThemeControlBackgroundBrush}">
-                        <Carousel Items="{Binding VisibleDockables}" SelectedItem="{Binding ActiveDockable}" IsVirtualized="False" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" />
-                    </Border>
-                </DockPanel>
+                        <Style Selector="TabItem:selected">
+                            <Setter Property="Background" Value="{DynamicResource ThemeControlBackgroundBrush}" />
+                            <Setter Property="Foreground" Value="{DynamicResource ApplicationAccentBrushMed}" />
+                            <Setter Property="BorderThickness" Value="1 0 1 1" />
+                            <Setter Property="Margin" Value="0 -1 0 0" />
+                            <Setter Property="Padding" Value="4 2 4 0" />
+                        </Style>
+                        <Style Selector="TabItem:selected:pointerover">
+                            <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}" />
+                            <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderLowBrush}" />
+                        </Style>
+                    </Panel.Styles>
+                    <Panel Name="DropPin" id:DockProperties.IsDropArea="True" Background="Transparent">
+                        <TabControl id:DockProperties.IsDropArea="True" Background="Transparent" Items="{Binding VisibleDockables}" SelectedItem="{Binding ActiveDockable, Mode=TwoWay}" TabStripPlacement="Bottom">
+                            <TabControl.ContentTemplate>
+                                <DataTemplate>
+                                    <Border Margin="0,0,0,-4" BorderThickness="1,0,1,1" BorderBrush="{DynamicResource ThemeBorderLowBrush}">
+                                        <ContentControl id:DockProperties.IsDropArea="True" Content="{Binding .}" IsVisible="{Binding IsExpanded}" x:CompileBindings="False"/>
```

Comment:
Why not Carousel ?

Reply:
doesnt Carousel take multiple `Items` rather than a single `Content`?

Reply:
Previously it was Carousel